stacks/Api.ts

```diff
@@ -201,6 +233,8 @@ export function ApiStack({ stack }: StackContext) {
             "GET /area-codes": areaCodeFunction,
             "GET /admin-areas": adminAreasFunction,
             "POST /street-manager": postStreetManagerFunction,
+            "GET /roadworks": roadworksFunction,
+            "GET /roadwork/{permitReferenceNumber}": roadworkByIdFunction,
```

Comment:
```suggestion
            "GET /roadworks/{permitReferenceNumber}": roadworkByIdFunction,
```
Keep it as roadworks to stay inline with the RESTful approach

Reply:
done